### ISSUE TITLE
Decompiling experimental GcGraphicsGlobals results in some corrupted values

### ISSUE THREAD

**Issue by trevizor** [monkeyman192/MBINCompiler#118]
Decompiling experimental/vulkan GCGRAPHICSGLOBALS.GLOBAL using the 1.77 MBINCompiler results in values such as:

```
<Property name="MinPixelSizeOfObjectsInShadowsSpace" value="1.345247E-43" />
  <Property name="MinPixelSizeOfObjectsInShadowsPlanet" value="6.726233E-44" />
  <Property name="MinPixelSizeOfObjectsInShadowsCockpitOnPlanet" value="3.601337E-43" />
  <Property name="MotionBlurThresholdOnFoot" value="1.401298E-45" />
  <Property name="MotionBlurThresholdInVehicle" value="1.401298E-45" />
  <Property name="MotionBlurThresholdSpace" value="1.401298E-45" />
  <Property name="MotionBlurThresholdDefault" value="1.793662E-42" />
```

**Comment by monkeyman192** [comment]
Sorry, hadn't got to it yet! Will get to it tomorrow sorry...

**Comment by monkeyman192** [comment]
This should be fixed in d218cb2597bfc17529060de0465d1f854a308d5d
